### PR TITLE
Adding duckieTV v1.1.3 standalone pkg

### PR DESCRIPTION
**Pull request by bibnumber**
# adding cask for duckietv
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There was a PR template when you submitted. You need to fill it.

**Comment by bibnumber** [comment]
Put the the template back in and double checked the list. Should be eligible now.

File: Casks/duckietv.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  sha256 '4c4ee965cf39db6333dad65fe2b56ebf412af490a1b091487ef5d1fe8989b359'
+
+  url "https://github.com/SchizoDuckie/DuckieTV/releases/download/#{version}/DuckieTV-#{version}-OSX-x64.pkg"
+  appcast 'https://github.com/SchizoDuckie/DuckieTV/releases.atom',
+          checkpoint: 'f11c61dde34cfe2a9f3e7150122340aa3f6680a634e00851066fccd4668e0103'
+  name 'duckieTV'
+  homepage 'https://schizoduckie.github.io/DuckieTV/'
+  license :oss
+
+  container type: :pkg
```
Comment (adidalal) [inline Casks/duckietv.rb]: Is this needed? @vitorgalvao

**Comment by bibnumber** [comment]
Okay i hate to bother… but I'm slightly confused how squashing works now.
Is it possible to squash this on github within this pull request?
or
Do i close this pull make the second commit locally again  squash it and push it to my fork and open a new pull request?

Sorry I'm pretty much a novice to git.

**Comment by vitorgalvao** [comment]
We can do the squashing for you.

File: Casks/duckietv.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+cask 'duckietv' do
+  version '1.1.3'
+  sha256 '4c4ee965cf39db6333dad65fe2b56ebf412af490a1b091487ef5d1fe8989b359'
+
+  url "https://github.com/SchizoDuckie/DuckieTV/releases/download/#{version}/DuckieTV-#{version}-OSX-x64.pkg"
```
Comment (vitorgalvao) [inline Casks/duckietv.rb]: Needs a `# github.com/SchizoDuckie/DuckieT was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Pull request merged**